src/value.rs

```diff
@@ -1,20 +1,43 @@
-use std::ops::{Add, Div, Mul, Sub};
+use derivative::Derivative;
```

Comment:
I wanted to ask, why did you decide to add this crate? I noticed that you along with it you added the Default trait to the Value. If this was the only purpose, then I would consider using the Default trait provided by the std library.

https://doc.rust-lang.org/std/default/trait.Default.html

ps
I do not mind at all =) however, I generally like to keep the dependency list as short as possible.

Reply:
I saw both options, but the version from std lib needs a few more lines to implement. Also I wanted to understand those traits a bit better. But I agree with you. I’ll change it.